spec/sum.test.js

```diff
@@ -0,0 +1,5 @@
+const sum = require('./sum');
+
+test('adds 1 + 2 to equal 3', () => {
+  expect(sum(1, 2)).toBe(3);
+});
```

Comment:
Nice work verifying that Jest was working! No need to ship this to production though. Delete this file.